src/index.ts

```diff
@@ -11,7 +11,7 @@ import { readdir } from "fs/promises";
 import { castArray } from "./utils.ts";
 import type { Listener } from "./types/listener.ts";
 
-const client = new Client({
+export const client = new Client({
```

Comment:
Unecessary at this point?

Reply:
Ah true, I should have emoved it in the merge resolve